.circleci/config.yml

```diff
@@ -37,8 +36,6 @@ jobs:
             - node_modules
             - playground/node_modules
           key: v1-dependencies-{{ checksum "yarn.lock" }}-{{ checksum "playground/yarn.lock" }}
-      # run linter
-      - run: yarn lint
```

Comment:
what happens if someone commits directly from github? If the file is not linted we might want to lint anyway here? Also could happen on merge commits maybe?

Reply:
Hmm, good point. I've split `eslint` and `prettier` out. Both are run on `lint-staged`, but only `eslint` in CI. Build is now working and committing files will still both lint and prettify them :)